tfx/extensions/google_cloud_big_query/example_gen/executor.py

```diff
@@ -65,9 +68,18 @@ def _BigQueryToExample(pipeline: beam.Pipeline, exec_properties: Dict[str, Any],
   """
   project = utils.parse_gcp_project(exec_properties['_beam_pipeline_args'])
   converter = _BigQueryConverter(split_pattern, project)
+  bq_kwargs = json.loads(exec_properties['custom_config'])
+
+  # if not bq_kwargs:
+  #   bq_kwargs={'dummy':None}
 
   return (pipeline
-          | 'QueryTable' >> utils.ReadFromBigQuery(query=split_pattern)
```

Comment:
I understand this is a good approach to get rid of redundant dependencies, but we have another code reference of `utils.ReadFromBigQuery` internally. Would you mind to make retain it and add a new argument name `big_query_kwargs`?

Reply:
Or, if you just retain this function, I will modify the internal references. I agree this is redundant code, too.

Reply:
@lego0901 I removed the other call of `utils.ReadFromBigQuery` for now

Reply:
I meant, there are internal codebase not exposed to the open source world, and some of them are calling the `utils.ReadFromBigQuery`. Therefore, it will cause some breakages to delete the function definition, so please keep it and I will handle it from my side :)